src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -267,17 +277,28 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             linkLabelHelp.Name = "linkLabelHelp"
             linkLabelHelp.TabStop = True
             '
-            'listViewExtensions
+            'DataGridViewExtensions
             '
-            listViewExtensions.AutoArrange = False
-            listViewExtensions.Columns.AddRange(New ColumnHeader() {colHeaderExtensionName, colHeaderExtensionVersion, colHeaderExtensionDescription})
-            resources.ApplyResources(listViewExtensions, "listViewExtensions")
-            listViewExtensions.FullRowSelect = True
-            listViewExtensions.HideSelection = False
-            listViewExtensions.Name = "listViewExtensions"
-            listViewExtensions.ShowItemToolTips = True
-            listViewExtensions.UseCompatibleStateImageBehavior = False
-            listViewExtensions.View = View.Details
+            '
+            'DataGridViewExtensions.AutoArrange = False
```

Comment:
Remove commented out code.

Reply:
Removed - to be clear, I left some commented code while I was waiting for review on the appearance/behavior before finalizing the PR.